Applications/utils/mem_usage.sh

```diff
@@ -16,30 +16,28 @@ log="\nPID - $process_id :"
 outlog=""
 count=0
 while true; do
-    if [ -d "/proc/$process_id" ]; then
-	output=`grep VmHWM /proc/$process_id/status`
-	read -ra mem_str <<< "$output"
-	count=$((count + 1))
-	mem_usage=$((${mem_str[1]}))
-	cpu_output=""
-	n=`nproc`
-	i=0
-	while [ "$i" -lt $n ]
-	do
-	    cpu_freq=`cat /sys/devices/system/cpu/cpu$i/cpufreq/scaling_cur_freq`
-	    cpu_output="$cpu_output $cpu_freq"
-	    i=$(expr $i + 1)
-	done
-	outlog="$outlog$count $mem_usage kB $cpu_output\n"
-	let mem_peak='mem_usage > mem_peak ? mem_usage : mem_peak'
-	sleep 1s # configure the sleep time if the program does not run long enough
-    else
-	break
-    fi
+	if [ -d "/proc/$process_id" ]; then
+		count=$((count + 1))
+		mem_usage=`cat /proc/$process_id/smaps | grep -E "Rss" | awk '{sum += $2;}END{print sum;}'`
```

Comment:
Could you leave the comment why we change ```$process_is/status``` to ```smaps``` ??

Reply:
The commnet is added.